kotlintest-runner/kotlintest-runner-jvm/src/main/kotlin/io/kotlintest/runner/jvm/TestEngineListener.kt

```diff
@@ -43,10 +43,7 @@ interface TestEngineListener {
   /**
    * Is invoked if a [Spec] throws an exception during initialisation
    */
-  fun specInitialisationFailed(klass: KClass<out Spec>, t: Throwable) {
-    beforeSpecClass(klass)
-    afterSpecClass(klass, t)
-  }
+  fun specInitialisationFailed(klass: KClass<out Spec>, t: Throwable) {}
```

Comment:
This function is now empty, and it seems final.
Is this on purpose?

Reply:
It's in an interface so it's open by default. It's got a body so you don't have to implement it unless you want to.